src/viadot/orchestration/prefect/tasks/bcp.py

```diff
@@ -1,34 +1,37 @@
-"""Tasks for bulk copying data from a CSV file into an SQLServer table using BCP."""
+"""Task for running BCP shell command."""
 
-from typing import Literal
+import subprocess
+from typing import Any, Literal
 
 from prefect import task
 
 from viadot.config import get_source_credentials
 from viadot.orchestration.prefect.exceptions import MissingSourceCredentialsError
-from viadot.orchestration.prefect.utils import get_credentials, shell_run_command
+from viadot.orchestration.prefect.utils import get_credentials
 
 
 @task
 def bcp(
-    path: str | None = None,
-    schema: str | None = None,
-    table: str | None = None,
+    path: str,
+    schema: str,
+    table: str,
     chunksize: int = 5000,
     error_log_file_path: str = "./log_file.log",
     on_error: Literal["skip", "fail"] = "skip",
     credentials_secret: str | None = None,
     config_key: str | None = None,
+    credentials: dict[str, Any] | None = None,
```

Comment:
Let's not expose `credentials` in tasks/flows since Prefect stores parameter values in plain text and can also display them in logs, which is a security issue.